tracpro/contacts/migrations/0006_contact_temba_modified_on_preset_data.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+from temba_client.base import TembaAPIError
+from dash.orgs.models import Org
+
+from ..models import Contact
+
+
+def no_op(apps, schema_editor):
```

Comment:
New in Django 1.8: https://docs.djangoproject.com/en/1.8/ref/migration-operations/#django.db.migrations.operations.RunPython.noop